### ISSUE TITLE
[Suggestion] Slower Rewind speed

### ISSUE THREAD

**Issue by Waldkoenig** [libretro/RetroArch#11049]
I know we can limit how often the Rewind feature takes a snapshot and if it would be set at for example every frame than that would be how fast we would rewind.

I think Rewind needs a delay setting that freezes a rewound frame for X milliseconds. That way we could rewind slower without having to force a snapshot every single frame.

So instead of Rewind rewinding as fast as possible - 1 snapshot per frame - it would rewind 1 snapshot, wait 100 milliseconds while the Rewind key is held and then rewind another snapshot and so on. 

It would make Rewind much more useful and resource friendly.

**Comment by bslenul** [comment]
Yeah atm it's almost impossible to rewind frame by frame without using macros :p 

I don't think pausing every frame (or whatever value you set in the Rewind menu) would be a good idea as it would get quickly annoying if you have to rewind far, but a pause just a single time after you hold the key would be great. Basically it could work like when you're holding a key on your keyboard with Notepad opened, I mean holding "X" doesn't write "XXXXXX" directly on your screen, first you get a single "X", then a slight pause and only then it starts spamming "XXXXXX".

This way if you want to rewind far you can simply hold the key just like before and you'll only get a small pause at the beginning, but if you need a really precise rewind you could just tap the hotkey.
